pkg/mark/mark.go

```diff
@@ -32,8 +32,25 @@ func ResolvePage(
 		return nil, page, nil
 	}
 
+	// check to see if home page is in Parents
+	homepage, err := api.FindHomePage(meta.Space)
+	if err != nil {
+		return nil, nil, karma.Format(
+			err,
+			"can't obtain home page from space %q",
+			meta.Space,
+		)
+	}
+
+	skip_home_ancestry := false
```

Comment:
Could you convert skip_home_ancestry and is_homepage to camelCase? In Go we don't really use snake_case at all :)

Reply:
Oh, thanks for telling me :) I'm from the Python world lol